front/package.json

```diff
@@ -3,40 +3,41 @@
   "license": "MIT",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve -H 0.0.0.0",
+    "start": "ng serve --host 0.0.0.0",
     "build": "ng build",
-    "build-prod": "ng build --prod --aot=false",
+    "build-prod": "ng build --prod --aot=false --build-optimizer=false",
```

Comment:
perhaps you can remove the option --aot=false now. It was a bug introduces in angular 5 if I remember well. So if you do you don't need to set the option --build-optimizer to false

Reply:
Fix in this commit:
https://github.com/Crunchy-Torch/coddy/pull/56/commits/f61e26b9bb4f5cf08e95c345772d9090fb72da92